migrations/versions/c0591105be81_adds_planet_model.py

```diff
@@ -0,0 +1,34 @@
+"""adds Planet model
+
+Revision ID: c0591105be81
+Revises: 
+Create Date: 2021-11-02 14:22:57.158289
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'c0591105be81'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('planet',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('name', sa.String(), nullable=True),
+    sa.Column('description', sa.String(), nullable=True),
+    sa.Column('type', sa.String(), nullable=True),
```

Comment:
Notice that since your model definitions didn't say that these fields were required, the migration marks them as being OK to be null.